api/pob/builder/v1/genesis.pulsar.go

```diff
@@ -455,7 +455,6 @@ var (
 	fd_Params_max_bundle_size          protoreflect.FieldDescriptor
 	fd_Params_escrow_account_address   protoreflect.FieldDescriptor
 	fd_Params_reserve_fee              protoreflect.FieldDescriptor
-	fd_Params_min_buy_in_fee           protoreflect.FieldDescriptor
```

Comment:
Why are we removing this? 

I think it _should_ be gone but I'm surprised it's happening in this PR

Reply:
It looks like there's not a diff in the corresponding protos, so it seems like @davidterpay @nivasan1 just didn't regen these files as part of cutting the previous release. 

This should probably happen as a part of CI

Reply:
Filed https://linear.app/skip/project/pob-65f1cb76c755/ENG